src/commands/infra/generate.ts

```diff
@@ -244,22 +243,51 @@ export const validateRemoteSource = async (
   logger.info(
     `Checking if source repo: ${safeLoggingUrl} has been already cloned to: ${sourcePath}.`
   );
-
   try {
-    // Check if .git folder exists in ${sourcePath}, if not, then clone
-    // if already cloned, 'git pull'
-    if (fs.existsSync(path.join(sourcePath, ".git"))) {
-      await gitFetchPull(sourcePath, safeLoggingUrl);
-    } else {
-      await gitClone(source, sourcePath);
```

Comment:
[change request] Can you please remove `gitClone` function from this file and also its unit tests.